app/assets/stylesheets/groceries/_show.css.scss

```diff
@@ -127,7 +127,7 @@
     text-align: center;
   }
 
-  #recipes {
+  #recipes, #location {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line